api/e2e/test/helpers_api_test.go

```diff
@@ -6,13 +6,13 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/gojek/turing/api/turing/service"
```

Comment:
Same comments as api/e2e/test/01_create_router_test.go.

Why has this been changed here? Can we run `goimports` on this file? May be the `make fmt` target can be adjusted to include this directory.